tuna/import_db.py

```diff
@@ -212,8 +211,9 @@ def print_sqlite_rows(cnx, cfg_filter):
 def main():
   """main"""
   args = parse_args()
-  args.table_cfg = ConvolutionConfig
-  args.table_perf_db = ConvolutionFindDB
+  dbt = DBTables(session_id=args.session_id)
```

Comment:
i think i wrote DBTable to take in a null value for session_id as well. I think it would be more clear if you can:
dbt = DBTables(session_id=None)